### ISSUE TITLE
Typescript Support

### ISSUE THREAD

**Issue by YassinEldeeb** [ahmadawais/create-node-cli#9]
A lot of devs rn are using Typescript and this boilerplate doesn't suit typesafe enthusiasts, unfortunately.

**Comment by ThaFishDance** [comment]
> A lot of devs rn are using Typescript and this boilerplate doesn't suit typesafe enthusiasts, unfortunately.


I agree. Perhaps a flag when running the initial setup 'npx create-node-cli --typescript'
